File: HELLO_ANDROID01/app/src/main/java/kr/co/aiai/app/MainActivity02.java
package kr.co.aiai.app;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity02 extends AppCompatActivity {

    TextView tv;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main02);

        Button btn = findViewById(R.id.btn);
        tv = findViewById(R.id.tv);

        btn.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view) {
                myclick();
            }
        });


    }

    void myclick(){
        String a = (String)tv.getText();
        Log.d("juho",a);
        int aa = Integer.parseInt(a);
        aa--;
        tv.setText(aa+"");
    }
}

File: HELLO_AI4_kotlin/ANDROID_kotlin/app/src/main/java/kr/co/aiai/myapp/MainActivity02.kt
package kr.co.aiai.myapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.TextView

class MainActivity02 : AppCompatActivity() {

    lateinit var tv:TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main02)
        val btn = findViewById<Button>(R.id.btn)
        tv = findViewById<TextView>(R.id.tv)

        btn.setOnClickListener(){
            myClick()
        }
    }

    fun myClick(){
        var a : String = tv.text as String
        var aa : Int = a.toInt()
        aa--
        tv.text = aa.toString()
    }

}